db/data_migration/20160512150912_republish_corporate_information_pages.rb

```diff
@@ -0,0 +1,7 @@
+CorporateInformationPage
+  .published
+  .includes(:document)
+  .(:document, :translations)
```

Comment:
Although, in this case it probably would have broken with a `NoMethodError: undefined method 'call'`.